filebeat/prospector/prospector_log.go

```diff
@@ -182,11 +180,9 @@ func (p *ProspectorLog) getFiles() map[string]os.FileInfo {
 func (p *ProspectorLog) matchesFile(filePath string) bool {
 	for _, glob := range p.config.Paths {
 
-		if runtime.GOOS == "windows" {
-			// Windows allows / slashes which makes glob patterns with / work
-			// But for match we need paths with \ as only file names are compared and no lookup happens
-			glob = strings.Replace(glob, "/", "\\", -1)
-		}
+		// Path and glob are cleaned to ensure we always compare clean paths
+		glob = filepath.Clean(glob)
+		filePath = filepath.Clean(filePath)
```

Comment:
It looks like cleaning `filePath` could be done only once if it were outside of the `for` loop.